src/net.h

```diff
@@ -63,6 +63,9 @@ static const int MAX_OUTBOUND_FULL_RELAY_CONNECTIONS = 8;
 static const int MAX_ADDNODE_CONNECTIONS = 8;
 /** Maximum number of block-relay-only outgoing connections */
 static const int MAX_BLOCK_RELAY_ONLY_CONNECTIONS = 2;
+/** Maximum number of block-relay-only anchor connections */
+static constexpr int MAX_BLOCK_RELAY_ONLY_ANCHORS = 2;
```

Comment:
This constant is only used internally within net.cpp, so I think it should be moved there, rather than be part of net's interface to other components.

Reply:
Thanks. [Updated](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-693330527).